applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebug.conf

```diff
@@ -23,6 +23,8 @@ CONFIG_ML_APP_ML_RUNNER_FRAME_SHIFT=10
 CONFIG_ML_APP_ML_STATE=y
 CONFIG_ML_APP_ML_STATE_CONTROL_BUTTON_ID=0x0000
 
+CONFIG_ML_APP_USB_STATE_ENABLE=y
```

Comment:
Can we use  `CONFIG_ML_APP_USB_STATE=y` here?
As I mentioned previously that would be consistent with other CAF and application-specific modules.

Reply:
yes, of course. I missed your point. I thought its about USB_STATE only, I'll fix that.

Reply:
So you still prefer to stick with _ENABLE ?

Reply:
@emob-nordic, please update it too

Reply:
fixed